packages/messages/.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
+  "env": {
+    "browser": true,
+    "commonjs": true,
+    "es2021": true,
+    "jest": true
+  },
+  "extends": [
+    "airbnb-base"
+  ],
+  "parserOptions": {
+    "ecmaVersion": "latest"
+  },  
```

Comment:
nit-pick: There are 2 spaces after `},`, can we remove them?

There are many vs-code extensions that helps you to see these kind of things. I use an extension called `Trailing Spaces` and it looks like this:
![Screen Shot 2022-03-16 at 15 27 54](https://user-images.githubusercontent.com/28733681/158701816-b599d0b3-3b93-49d7-94dc-da6c5a6325b1.png)

Reply:
Done